test/integration/patients/patient/dashboard.js

```diff
@@ -55,7 +64,20 @@ context('patient dashboard page', function() {
 
     cy
       .get('.patient__list')
-      .contains('Test Action')
+      .find('.table-list__item')
+      .first()
+      .next()
+      .should('contain', 'Second In List');
+
+    cy
+      .get('.patient__list')
+      .find('.table-list__item')
+      .last()
+      .should('contain', 'Last In List');
+
+    cy
+      .get('.patient__list')
+      .contains('First In List')
```

Comment:
I think this should be `first()` `should('contain, 'First In List')`  `.next.should.next.should`